static/css/components/book.less

```diff
@@ -22,6 +22,9 @@
     a {
       margin: auto;
     }
+    a {
+      width: 100%;
+    }
```

Comment:
```suggestion
```
There's no need to create a new `a` selector here --- just use the one directly above this.

Reply:
Thanks for the feedback!
I am not so familiar with Less so might be misunderstanding.
Lines 19, 22, and 25 all add `a` selectors with a single property defined. Should they all be  combined or are the blocks defined on 19 and 22 different in some way?

Thanks!

Reply:
Lines 18-21 applies to `img` and `a` elements (the compiled selector is `.book .book-cover a, .book .book-cover img`).

Lines 22-24 only apply to `a` elements.

Here is an overview on how nesting works in Less: https://lesscss.org/#nesting

Reply:
Ah, thanks for the explanation and link! 
I combined them into a single selector.